components/home/Hero.vue

```diff
@@ -5,17 +5,28 @@
         class="d-flex flex-column align-items-center justify-content-center text-center my-5"
       >
         <h1 class="hero__heading">
-          Best <span class="hglight">No-Code</span> Form Builder
+          <span
+            v-for="item in title"
+            :key="item.id"
+            :class="{ hglight: item.highlight }"
+          >
+            {{ item.text }}
+          </span>
         </h1>
         <p class="hero__subheading mt-3">
-          Create elegant forms, surveys and quizzes your audience will love to
-          answer.
+          {{ description }}
         </p>
         <a
-          href="https://app.formester.com/users/sign_up"
+          v-if="button && button?.length > 0"
+          :href="button[0].link"
           class="btn button hero__button mt-4 mb-4 mb-md-5"
-          >Create Form for Free</a
+          :class="{
+            hero__button: button[0].type === 'Primary',
+            hero__invert__button: button[0].type === 'Secondary',
+          }"
         >
+          {{ button[0].text }}
+        </a>
```

Comment:
Same here we are getting button by index. And the variable name is also singular

Reply:
In hero section, we have only one button. So in the cms, the name is singular. So we have to access it as a singular variable.

Reply:
If we have only one button then why is it array?

Reply:
So if in the future we decide to add multiple buttons here, then we will need to remove all the data of that collection from the CMS, and then change the data type to support multiple buttons. Then again add data to that collection.

But Now since it already supports multiple buttons, we don't have to change anything on the cms side. We will only need to add it here (formester-site).

That is why I kept it multiple from the start.

Reply:
Then the variable name should be `buttons` and not `button` else it'll confuse the devs

Reply:
Okay got it. Will rename it.